lib/pages/image_viewer/image_viewer.dart

```diff
@@ -60,6 +75,28 @@ class ImageViewerController extends State<ImageViewer> {
     }
   }
 
+  void onDoubleTap() {
+    final position = tapDownDetails!.localPosition;
```

Comment:
please check null instead
```suggestion
    final position = tapDownDetails?.localPosition;
    if (position == null) return;
```